.gitignore

```diff
@@ -3,7 +3,22 @@ work/
 data/
 results/
 .DS_Store
-tests/
 testing/
 testing*
 *.pyc
+
+.local
+.config
+.bash_history
+
+R_sessionInfo.log
+
+dev/docker/static_reports/test_data/*
+dev/docker/static_reports/test_output/*
+!dev/docker/static_reports/test_data/_dummy.txt
+!dev/docker/static_reports/test_output/_dummy.txt
```

Comment:
Why escaping this file if not in the repo?

Reply:
Yep, I have cleaned up this file now